[2023-03-16 16:01:49,094] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: our_first_dag.first_task manual__2023-03-16T07:01:46.983263+00:00 [queued]>
[2023-03-16 16:01:49,098] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: our_first_dag.first_task manual__2023-03-16T07:01:46.983263+00:00 [queued]>
[2023-03-16 16:01:49,098] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-16 16:01:49,098] {taskinstance.py:1239} INFO - Starting attempt 1 of 4
[2023-03-16 16:01:49,098] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-16 16:01:49,103] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2023-03-16 07:01:46.983263+00:00
[2023-03-16 16:01:49,104] {standard_task_runner.py:52} INFO - Started process 98699 to run task
[2023-03-16 16:01:49,107] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag', 'first_task', 'manual__2023-03-16T07:01:46.983263+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/var/folders/x1/7qlvjnbj06q1nkyhy70vbb2c0000gn/T/tmp12hi72_2', '--error-file', '/var/folders/x1/7qlvjnbj06q1nkyhy70vbb2c0000gn/T/tmpkkpuontr']
[2023-03-16 16:01:49,108] {standard_task_runner.py:77} INFO - Job 5: Subtask first_task
[2023-03-16 16:01:49,128] {logging_mixin.py:109} INFO - Running <TaskInstance: our_first_dag.first_task manual__2023-03-16T07:01:46.983263+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2023-03-16 16:01:49,146] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=byeonghwa
AIRFLOW_CTX_DAG_ID=our_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-16T07:01:46.983263+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-16T07:01:46.983263+00:00
[2023-03-16 16:01:49,147] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/x1/7qlvjnbj06q1nkyhy70vbb2c0000gn/T
[2023-03-16 16:01:49,147] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hello world, this is the first task!']
[2023-03-16 16:01:49,157] {subprocess.py:85} INFO - Output:
[2023-03-16 16:01:49,164] {subprocess.py:89} INFO - hello world, this is the first task!
[2023-03-16 16:01:49,164] {subprocess.py:93} INFO - Command exited with return code 0
[2023-03-16 16:01:49,171] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, execution_date=20230316T070146, start_date=20230316T070149, end_date=20230316T070149
[2023-03-16 16:01:49,214] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-16 16:01:49,222] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
